Hermes/appscale/hermes/hermes_server.py

```diff
@@ -0,0 +1,83 @@
+""" Web server/client that polls the AppScale Portal for new tasks and
```

Comment:
Though the comment was already outdated, your changes made it even more outdated=)
I would suggest:
```
""" Web server providing different metrics of AppScale
nodes, processes and services.
"""
```